docs/cado/integrations/webhooks.md

```diff
@@ -120,6 +120,44 @@ You can create Jira tickets using Cado webhooks.
 
 ![Jira Webhook](/img/jira_webhook.png)
 
+More complex webhooks are available to the Detection webhooks, for example this can be set as the "Custom Payload" in Cado for the "Detection Updated" webhook:
+
+```json
+{
+  "detection_id": "$detection_id",
```

Comment:
One thing to note, users don't need to set this as we send it regardless. This being detection_id, event_type, and group_id.